.github/workflows/ci.yml

```diff
@@ -83,7 +84,18 @@ jobs:
         set -e
         set -x
         apt-get --quiet update
+
+        # Add linuxcnc.org deb archive to apt sources list.
+        apt-get --yes --quiet install gpg software-properties-common
+        gpg --keyserver hkps://keyserver.ubuntu.com --recv-key 3CB9FD148F374FEF
+        gpg --output /etc/apt/trusted.gpg.d/linuxcnc-deb-archive.gpg --export 3CB9FD148F374FEF
+        DIST=$(echo ${{matrix.image}} | cut -d : -f 2)
+        add-apt-repository "deb http://linuxcnc.org $DIST base"
+        apt-get --quiet update
+
+        # Install stuff needed to turn a linuxcnc working tree into a debian source package.
```

Comment:
This is equivalent to adding random stuff from the internet and allow anyone with the ability to hijack TCP connections to run anything they want on the CI test host.  Perhaps only enable it for oldstable and stable, and not for testing and unstable, to at least reduce the exposure as much as possible?  Or perhaps provide the GPG key alongside the CI script, to start the chain of trust within the git repository instead of having to trust the hkps server protocol and connection?

Reply:
That's a good idea, I'll include the pubkey in the repo.